jetty-websocket/websocket-client/src/main/java/org/eclipse/jetty/websocket/client/WebSocketClient.java

```diff
@@ -255,6 +272,24 @@ public WebSocketClient(final WebSocketContainerScope scope, EventDriverFactory e
         this.sessionFactory = sessionFactory == null ? new WebSocketSessionFactory(this) : sessionFactory;
     }
 
+    WebSocketClient(final HttpContainerScope httpContainerScope)
```

Comment:
Why is this only package private?

Reply:
I don't want this to be exposed to end users for one.
It's an internal concept.

Reply:
I meant why is this package private rather than just private?

Reply:
It was being used by the javax.websocket ClientContainer directly.
I'll move this to private now that the latest commit has removed HttpContainerScope.

Reply:
Latest commit makes this constructor private.